packages/turndown-plugin-gfm/src/tables.js

```diff
@@ -187,6 +187,12 @@ const nodeContains = (node, types) => {
   return false;
 }
 
+// Deskotp app table have, by default, border-collapse style,
+// so we need one more to know if the table is modified
+const isModifiedRTETable = (tableNode) => {
+  return tableNode.style.length > 1;
+}
```

Comment:
Currently, this could break if the renderer package is updated to apply different default styles to tables. Here are a few ideas to prevent regressions:
- Add a test that verifies that a table rendered with `@joplin/renderer` is converted back to Markdown (with `preserveNestedTables: true`).
- Would it be possible to check for the presence of specific style attributes? For example, convert to HTML if `['background-color', 'border-color', 'border']` includes one of the style properties.


Thank you for working on this!